executors/rs

```diff
@@ -1,3 +1,8 @@
 #!/bin/sh
+<<<<<<< HEAD
```

Comment:
There're conflict markers :) Anyway, I think we don't need this commit anymore. It seems you've added in 2014, but I have also added it in 2016.